smithy-rules-engine/src/main/java/software/amazon/smithy/rulesengine/traits/EndpointTestsTraitValidator.java

```diff
@@ -92,6 +176,18 @@ private List<ValidationEvent> validateOperationInput(
                 .startingContext("The operationInput value for an endpoint test "
                         + "does not match the operation's input shape")
                 .build();
-        return inputShape.accept(validator);
+
+        // Error test cases may use invalid inputs as the mechanism to trigger their error,
```

Comment:
I'm still not entirely convinced that this is valid for SDKs that implement operation input tests. If a test's inputs are invalid, it seems likely to trigger an error in the SDK outside of the endpoint resolver and would therefore not match the error specified by the test?

Reply:
This isn't changing from before, it's just moving to clean up the `validate` method above.